Build script flakiness

Sometimes the build script fails on the deploy step, complaining that the `dist/site` directory does not exist. I can think of two possible reasons:
* For some reason either Loconotion isn't outputting where it should, or;
* The Docker volume is not mounting as it should.

Same thing here

deploy.sh logs...

```
Run git config --global user.name "github-actions[bot]"
  git config --global user.name "github-actions[bot]"
  git config --global user.email "99999+github-actions[bot]@users.noreply.github.com"
  
  GIT_DEPLOY_DIR=dist/site \
  GIT_DEPLOY_BRANCH=gh-pages \
  GIT_DEPLOY_REPO="https://***@github.com/xxxxxxxxxx/notion-website.git" ./deploy.sh
  shell: /usr/bin/bash -e {0}
Deploy directory 'dist/site' does not exist. Aborting.
Error: Process completed with exit code 1.
```

--- a/.github/workflows/main.yml
+++ b/.github/workflows/main.yml
@@ -25,14 +25,11 @@ jobs:
         
       - name: Run Loconotion
         run: |
-          docker-compose run \
+          docker run \
           -v "$GITHUB_WORKSPACE/pages_repo/dist:/app/loconotion/dist" \
           -v "$GITHUB_WORKSPACE/pages_repo/site.toml:/app/loconotion/site.toml" \
           loconotion "site.toml"
         working-directory: loconotion
-      - name: What's the deal?
-        run: |
-          find "$GITHUB_WORKSPACE"
       - name: Push to GitHub pages
         run: |
           git config --global user.name "github-actions[bot]"

